recipes/yq/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "yq" %}
+{% set version = "2.4.1" %}
+{% set sha256 = "4833d4055b0f1c1f1a2fd292421b3472da39c7dc2727d7819efd11065a5fd310" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: 
+    - python -m pip install --no-deps --ignore-installed .
+    - chmod 755 $PREFIX/bin/{y,x}q
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - pyyaml >=3.11
+    - xmltodict >=0.11.0
+    - jq
+
+test:
+  requires:
+    - posix  # [win]
+  imports:
+    - yq
+  commands:
+    - yq --help
+    - xq --help
+    - bash -c "test $(cat $RECIPE_DIR/meta.yaml | yq -r .package.name) = 'yq'"
+
+about:
+  home: https://github.com/kislyuk/yq
+  license: Apache-2.0
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Upstream is not including the License file in src package

Reply:
Could you please raise an issue to ask them if they could include the license?

Reply:
I opened the issue: c.f. kislyuk/yq#24